candishared/database_update_tools.go

```diff
@@ -84,6 +88,7 @@ type DBUpdateTools struct {
 	KeyExtractorFunc    func(structTag reflect.StructField) (res DBUpdateOptionKeyExtractorResult)
 	FieldValueExtractor func(reflect.Value) (val any, skip bool)
 	IgnoredFields       []string
+	fieldType           string
```

Comment:
The new fieldType in DBUpdateTools struct is added but not used. Should it be removed or is it for future use?